### PR TITLE
Support specifying duration for bootstrap OTP, improve logging.

### PR DESCRIPTION
**Pull request by rgooch**
This also changes the endpoint path to be properly camelCased.
Add more comments.

### CODE REVIEW

File: cmd/keymasterd/adminHandlers.go [hunk 1]
```diff
@@ -164,54 +168,80 @@ func (state *RuntimeState) deleteUserHandler(
...
+			err.Error())
 		return
 	}
 	profile.BootstrapOTP = bootstrapOTP
 	err = state.SaveUserProfile(username, profile)
 	if err != nil {
-		logger.Printf("error saving profile randr=%s", err)
-		state.writeFailureResponse(w, r, http.StatusInternalServerError, "")
+		state.logger.Printf("error saving profile randr=%s", err)
+		state.writeFailureResponse(w, r, http.StatusInternalServerError,
```
Comment (cviecco) [inline cmd/keymasterd/adminHandlers.go]: We never send the actual error back to the client when the error is internal. This is a source of security issues.

File: cmd/keymasterd/adminHandlers.go [hunk 2]
```diff
@@ -164,54 +168,80 @@ func (state *RuntimeState) deleteUserHandler(
...
+			"Duration over 1 day not allowed")
+		return
+	}
+	bootstrapOTP := bootstrapOTPData{ExpiresAt: time.Now().Add(duration)}
 	bootstrapOTP.Value, err = genRandomString()
 	if err != nil {
-		logger.Printf("error generating randr=%s", err)
-		state.writeFailureResponse(w, r, http.StatusInternalServerError, "")
+		state.logger.Printf("error generating randr=%s", err)
+		state.writeFailureResponse(w, r, http.StatusInternalServerError,
```
Comment (cviecco) [inline cmd/keymasterd/adminHandlers.go]: We never send the actual error on internal errors

File: cmd/keymasterd/adminHandlers.go [hunk 3]
```diff
@@ -164,54 +168,80 @@ func (state *RuntimeState) deleteUserHandler(
...
+		duration, err = time.ParseDuration(formDuration[0])
+		if err != nil {
+			state.writeFailureResponse(w, r, http.StatusBadRequest, err.Error())
+			return
+		}
+	}
+	if duration < time.Minute {
+		duration = time.Minute
 	}
+	if duration > time.Hour*24 {
```
Comment (cviecco) [inline cmd/keymasterd/adminHandlers.go]: make it a const

File: cmd/keymasterd/adminHandlers.go (hunk 1 above)
Comment (rgooch) [reply cmd/keymasterd/adminHandlers.go]: Fixed.

File: cmd/keymasterd/adminHandlers.go (hunk 2 above)
Comment (rgooch) [reply cmd/keymasterd/adminHandlers.go]: Fixed.

File: cmd/keymasterd/adminHandlers.go (hunk 3 above)
Comment (rgooch) [reply cmd/keymasterd/adminHandlers.go]: Done.

**Pull request merged**